index.html

```diff
@@ -1,52 +1,314 @@
+						<div class="tags">
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
+						<div class="project_link">
+							<p>
+								<a href="#">See Project</a>
+							</p>
+						</div>
+					</div> 
+				</div>
+
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio (2).png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Tonic</p>
+							<div class="work_history">
+								<p class="client">
+									<span class="client_name">Canopy</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="role">
+									<span class="role_name">Back End Dev</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="year">
+									<span class="date">2015</span>
+								</p>
+							</div>   
+						</div>       		
+						<p class="work_summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+						<div class="tags">
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
+						<div class="project_link">
+							<p>
+								<a href="#">See Project</a>
+							</p>
+						</div>
+					</div> 
+				</div>
+
+				<div class="work_item">
+					<img src="images/Snapshoot Portfolio (3).png" alt="snapshot portifolio image">
+					<div class="work_details">
+						<div class="project">
+							<p class="work_item_header">Multi-Post Stories</p>
+							<div class="work_history">
+								<p class="client">
+									<span class="client_name">Canopy</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="role">
+									<span class="role_name">Back End Dev</span>
+								</p>
+								<p class="counter">
+									<img src="images/Counter.png" alt="counter icon">                            
+								</p>
+								<p class="year">
+									<span class="date">2015</span>
+								</p>
+							</div>   
+						</div>       		
+						<p class="work_summary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+						<div class="tags">
+							<p class="html">
+								<span>html</span>
+							</p>
+							<p class="css">
+								<span>css</span>
+							</p>
+							<p class="js">
+								<span>javascript</span>
+							</p>
+						</div>
```

Comment:
- :red_circle: The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be wrapped inside a list ul and each item of them in a li tag. Please replace the `div` container with a `ul` container